### ISSUE TITLE
Allow cwebp binary path to be set through environment variable

### ISSUE THREAD

**Issue by rosell-dk** [rosell-dk/webp-convert#184]
The option was requested in #92

**Comment by rosell-dk** [comment]
These can now be set with CWEBP_PATH

To set environment variables in Apache, you can add a line like this in your `.htaccess` or vhost configuration:
```
SetEnv CWEBP_PATH /usr/bin/cwebp
```

To find out where the cwebp binary is, open a console and type:
```
whereis cwebp
```

**Comment by rosell-dk** [comment]
NOTE NOTE NOTE:
I have changed the name of the environment variable to "WEBPCONVERT_CWEBP_PATH" in the last commit (will be released as 2.0.3). I'm aware that this is an API change and it should trigger a major version shift, - but I am not doing that, hoping that only few has yet migrated to 2.0 and only few are using this feauture

Btw, you can now also set it with a contant:
```php
define('WEBPCONVERT_CWEBP_PATH', '/usr/bin/cwebp');
```
